ardrone_driver_node:
    ros__parameters:
        timeout: 0.03              # timeout for socket in lib for send/get message
        watchdog_interval: 0.03               # timeout for socket in lib for video
        # NAV data for config AR.Drone 2.0
        # 'demo' is need for working NAV data
        # ***'metadata' and 'vision_detect' always included in 'demo'***
        # navdata_options: "{'demo': True,'time': True,'raw_measures': True,'pressure_raw': True,'magneto': True,'euler_angles': True,'pwm': True}"
        navdata_options: "{'demo': True}"
        # navdata_options: "{'demo': False}"
        # full availible NAV data options you can see in documantation for pyardrone
